integration/src/main/resources/configs/perf/GetStatus.json

```diff
@@ -0,0 +1,17 @@
+{
+  "name": "GetStatus",
+  "description": "Call the service status multiple times. No authentication required.",
+  "serverSpecificationFile": "catalog-local.json",
```

Comment:
Should this be `catalog-perf.json`?

Reply:
Yes, thanks! (Not sure why there isn't a way to set this across all test scripts in a single env, though)

Reply:
Oh! Actually we can override this so we could just have one set of configs instead of duplicating them per environment: https://github.com/DataBiosphere/terra-test-runner/blob/main/README.md#override-the-server-from-the-command-line